Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -584,6 +522,26 @@ namespace EMotionFX
         Update(nullptr, nullptr, PhysicsSetup::ColliderConfigType::Unknown, AzPhysics::ShapeColliderPairList(), nullptr);
     }
 
+    void ColliderContainerWidget::SetFilterString(AZStd::string str)
+    {
+        for (auto* widget : m_colliderWidgets)
+        {
+            widget->SetFilterString(str);
+        }
+    }
+
+    bool ColliderContainerWidget::HasVisibleColliders() const
+    {
+        return m_colliderWidgets.size() > 0 &&
```

Comment:
update: Please use AZStd::any_of (and the size check is likely redundant too as `any_of` will fail if size is 0)